docs/iris/src/whatsnew/3.0.4.rst

```diff
@@ -0,0 +1,592 @@
+.. include:: ../common_links.inc
+
+v3.0.4 (?? ??August 2021)
+*************************
+
+This document explains the changes made to Iris for this release
+(:doc:`View all changes <index>`.)
+
+
+.. dropdown:: :opticon:`alert` v3.0.4 Patches
+   :container: + shadow
+   :title: text-primary text-center font-weight-bold
+   :body: bg-light
+   :animate: fade-in
+   :open:
+
+   The patches included in this release include:
+
+   🐛 **Bugs Fixed**
+
+   #. `@lbdreyer`_ modified :meth:`~iris.cube.Cube.intersection` to use a tolerant 
+      equality check, when looking for cells that straddle the wrapping point.
+      (:pull:`4220`)
+
+
```

Comment:
Missing a `v3.0.3` dropdown, so I'm guessing this file wasn't based originally on [`3.0.3.rst`](https://github.com/SciTools/iris/blob/82dcba7fb6e33f70069800f7a40c3ae6171f522f/docs/iris/src/whatsnew/3.0.3.rst). Can you address this?

Reply:
Not at all, it **_was_** based on 3.0.3, but I didn't understand what I was doing !

I'm now seeing that the different entries are aranged into a section for each point release,
but there is only one "v3.x.x (date)" subheading at the top.

Hope this is now correct ?